sample-node-config/aws/README.md

```diff
@@ -0,0 +1,234 @@
+# Example Hydra Node Infrastructure
+
+This directory contains some [Terraform](https://www.hashicorp.com/products/terraform) and AWS based infrastructure code to setup a single [Hydra node](https://hydra.family/head-protocol/docs/getting-started/installation) connected to a [Cardano node](https://docs.cardano.org/getting-started/installing-the-cardano-node) running on `preview` testnet. It's not a complete turnkey solution and requires some tweaking and parameterisation to be completely usable but we thought it would be good starting point for new Hydra users.
+
+## Configuring Hydra Node
+The [configuration script](./scripts/configure-testnet.sh) assumes this image must be a reasonably recent Ubuntu/Debian Linux distribution, with [docker](https://docker.io) and [docker-compose](https://docs.docker.com/compose/) installed, and a configured user `ubuntu` with `sudo` access.
+
+The configuration process expects to find some files which are not provided by default and which are required for starting the Hydra node:
+* A Hydra signing key file `hydra-key.sk` which will be used in the Head to sign snapshots.
```

Comment:
That's not clear at this point of the documentation for me if I must create all these files now or if it is something I'll do inside the VM later. Should I run the terraform stuff above and then go back to this section or am I supposed to do something right now?

Reply:
great point.
tbh, this part of the doc was taken from @abailly-iohk's.
i tried to adapt it to make sense but i never ended liking it.
will give it a second round and will ask you to let me know if its better!
ty 🙏